adapter/internal/operator/apis/dp/v1alpha1/apipolicy_types.go

```diff
@@ -78,6 +79,33 @@ type CORSPolicy struct {
 	AccessControlExposeHeaders    []string `json:"accessControlExposeHeaders,omitempty"`
 }
 
+// OPAPolicy holds OPA policy information
+type OPAPolicy struct {
+	// ServerURL is the URL of the OPA server
+	ServerURL string `json:"serverURL,omitempty"`
```

Comment:
please add validation regex for url

Reply:
Added by https://github.com/wso2/apk/pull/1193/commits/cd5afd7d2c0907424415d64317afb9ad9a5dc707